logging/nox.py

```diff
@@ -74,9 +74,10 @@ def system_tests(session, python_version):
     session.run(
         'py.test',
         '-vvv',
+        '-s',
         'tests/system.py',
         *session.posargs,
-        success_codes=range(0, 100),
+        success_codes=range(0, 100)
```

Comment:
I got syntax error when I run `nox -l` if I keep the comma, maybe because my environment does not support this syntax?